apps/e2e/cypress/support/userCommands.js

```diff
@@ -88,6 +89,16 @@ Cypress.Commands.add("getMe", () => {
   });
 });
 
+Cypress.Commands.add("getMeAllData", () => {
```

Comment:
```suggestion
Cypress.Commands.add("getMe", () => {
```

Reply:
That was deliberate as there is already a "getMe" command present that gathers only a slice of data returned by query `me`.
So rather than change existing commands, I introduced a new one that gathers all data when required.

Reply:
Would it be better to just update the `cy.getMe` to return all data instead of having two co-existing queries that do similar things with slightly different data? This smells of another `Pool` v `PoolAdvertisement` thing 😬

Reply:
`cy.getMe` runs query `me`

![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/40485260/4690c49a-0432-4eac-b15b-398d06b3d331)

Which will only ever return the below

![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/40485260/266a221c-99e3-426d-b293-881cd787974b)

What I added, runs query `getMe`

![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/40485260/e6761df9-75b4-483f-8779-6983d4fb990a)

Which returns all

![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/40485260/2d111254-1c64-43f8-bd6f-01e20f56071d)

Reply:
So really, they are two distinct queries that accomplish different things, if anything, `cy.getMe` should be `cy.me` to match the query to the command.

Reply:
It's probably good to not overfetch a bunch of unneeded data.  Perhaps better naming of the queries would help clarify the difference?

Reply:
Then again, for e2e tests, overfetching doesn't really matter.  I think I prefer Eric's recommendation of switching the single command to use the bigger query.

Reply:
done in https://github.com/GCTC-NTGC/gc-digital-talent/pull/6756/commits/72b329cc25ee5036c52e2d9c0e9729b77ca4b0c0